vm/systemSmartContracts/auction.go

```diff
@@ -125,11 +125,13 @@ func (s *stakingAuctionSC) Execute(args *vmcommon.ContractCallInput) vmcommon.Re
 		return s.unJail(args)
 	}
 
+	s.eei.AddReturnMessage("invalid method to call")
 	return vmcommon.UserError
 }
 
 func (s *stakingAuctionSC) unJail(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
 	if len(args.Arguments) == 0 {
+		s.eei.AddReturnMessage("invalid number of arguments: got 0")
```

Comment:
You can use the same patter expected min %d, got %d

Reply:
done.